### PR TITLE
Switch from #\20 to #\9

### LINKED ISSUE #11: Consider using \9
`:not(#\9)`, where `\9` is a tab character, can be used. It works in the same way as `:not(#\20)` but is one character shorter.

Other alternatives are `\A`, `\C` and `\D`.
* https://html.spec.whatwg.org/multipage/dom.html#the-id-attribute
   (*The value must not contain any ASCII whitespace.*)
* https://infra.spec.whatwg.org/#ascii-whitespace
   (*An ASCII whitespace is U+0009 TAB, U+000A LF, U+000C FF, U+000D CR, or U+0020 SPACE.*)

### PR DESCRIPTION
**Pull request by valtlai**
Fixes #11

Switch from `#\20` to `#\9`

### CODE REVIEW

**Comment by iamstarkov** [comment]
imho, I don't see strong reasoning behind this.

**Comment by valtlai** [comment]
I thought why not use a shorter one and save some bytes. But I understand this has no practical effect.

File: index.js [hunk 1]
```diff
@@ -5,21 +5,21 @@ var objectAssign = require('object-assign');
...
 		overrideIds: true,
 		// The thing we repeat over and over to make up the piece that increases specificity
-		// > Consider use :not(#\20), :not(.\20) and :not(\20)
-		// > Rationale: \20 is a css escape for U+0020 Space, and neither classname, nor id, nor tagname can contain a space
+		// > Consider use :not(#\9), :not(.\9) and :not(\9)
+		// > Rationale: \9 is a css escape for U+0009 Character Tabulation, and neither classname, nor id, nor tagname can contain a tab character
 		// >
-		// > — https://twitter.com/subzey/status/829050478721896448
-		stackableRoot: ':not(#' + CSS_ESCAPED_SPACE + ')'
+		// > — https://twitter.com/subzey/status/829050478721896448 (originally \20)
```
Comment (MadLittleMods) [inline index.js]: @valtlai Let's remove this Twitter quotation now that things have changed and don't match up as nicely.

**Review by MadLittleMods — CHANGES_REQUESTED** [review]
Remove Twitter quotation and add an "Upcoming" changelog entry for these changes 😀

**Comment by MadLittleMods** [comment]
Thanks @valtlai 💾 📉 🚀

**Comment by MadLittleMods** [comment]
Published to npm, [`postcss-increase-specificity@0.5.0`](https://github.com/MadLittleMods/postcss-increase-specificity/blob/master/CHANGELOG.md#v050---2017-5-26) , https://www.npmjs.com/package/postcss-increase-specificity 💥

**Pull request merged**